parm/atm/obs/lists/gdas_prototype_4d.yaml.j2

```diff
@@ -0,0 +1,16 @@
+observers:
+{% filter indent(width=2) %}
+{% include 'atm/obs/config/amsua_n19.yaml.j2' %}
+{% include 'atm/obs/config/sondes.yaml.j2' %}
+{% include 'atm/obs/config/atms_n20.yaml.j2' %}
+{% include 'atm/obs/config/aircraft.yaml.j2' %}
+{% include 'atm/obs/config/satwind.yaml.j2' %}
+{% include 'atm/obs/config/omi_aura.yaml.j2' %}
+{% include 'atm/obs/config/ompsnp_npp.yaml.j2' %}
+{% include 'atm/obs/config/ompstc8_npp.yaml.j2' %}
+{% include 'atm/obs/config/cris-fsr_n20.yaml.j2' %}
+{% include 'atm/obs/config/cris-fsr_npp.yaml.j2' %}
+{% include 'atm/obs/config/sfc.yaml.j2' %}
+{% include 'atm/obs/config/sfcship.yaml.j2' %}
+{% set cycle = current_cycle | strftime("t%Hz") %}
```

Comment:
```suggestion
```
Remove `cycle` as it is not used in the template.

Reply:
Change committed at b174bf2